rails-commenting.rb

```diff
@@ -7,24 +7,29 @@
 # FILE: app/controller/blog_posts_controller.rb
 
 # ---1)
+# Create the controller with a class name of BlogPosts and a controller action method named index to list all available data from the database.
 class BlogPostsController < ApplicationController
   def index
     # ---2)
+    # Create an instance variable named posts that contains the object BlogPost and use the built-in .all method to enumerate over the elements contained within the object.
     @posts = BlogPost.all
   end
 
```

Comment:
For your notes and understanding: I suggest go back through questions 2-10 and make sure you discuss what the active record query will be returning from the database (all the entries or one entry). Also if there are any if/else/redirect discuss what happens for a true condition.